src/OpenTelemetry.Exporter.Console/ConsoleActivityExporter.cs

```diff
@@ -32,10 +32,16 @@ public override ExportResult Export(in Batch<Activity> batch)
         {
             foreach (var activity in batch)
             {
-                this.WriteLine($"Activity.Id:          {activity.Id}");
-                if (!string.IsNullOrEmpty(activity.ParentId))
+                this.WriteLine($"Activity.TraceId:          {activity.TraceId}");
```

Comment:
Activity.Id is what goes inside "traceparent" header... its a bit cleaner to display explicitly traceid,spanid etc.